tests/test_ec2/test_elastic_block_store.py

```diff
@@ -453,128 +453,175 @@ def test_snapshot_filters():
     )
 
 
-@mock_ec2_deprecated
```

Comment:
Please keep the existing test case in tact, to ensure we're still supporting the deprecated use case.

The snapshot_with_userid should be a separate test case

Reply:
I added the old test case back in, as well as rebased with master. Thanks!